doc/releases/release-notes-3.5.rst

```diff
@@ -100,14 +100,132 @@ Bluetooth
 
 * Audio
 
+  * Fixed bad size of metadata_backup in ASCS
+  * Fixed TMAS characteristic permissions
+  * Fixed BAP bt_bap_scan_delegator_find_state implementation
+  * Fixed possible ASCS ASE stuck in releasing state
+  * Fixed ASCS to drop ISO PDUs if ASE is not in streaming state
+  * Fixed audio stack to accept empty CCID list in audio metadata
+  * Renamed bt_codec to bt_audio_codec_{cap, conf, data}
```

Comment:
please try to use double backquotes for anything that should be formatted as code

```suggestion
  * Renamed ``bt_codec`` to ``bt_audio_codec_{cap, conf, data}``
```

Reply:
Is that the right way of doing it in RST?

In most RST files I've seen (and written), we've used 
:code:\`bt_codec\` instead

Reply:
Either way should be fine bust a vast majority of our docs uses double backticks (and that's what's also recommended in the [guidelines](https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#content-highlighting)).

Reply:
fixed